test/OpenSpaceToolkit/Physics/Data/Manifest/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "space_weather_CSSI": {
+        "path": "environment/atmospheric/earth/CSSISpaceWeather",
+        "filenames": "SW-Last5Years.csv",
+        "remote_sources": [
+            {
+                "url": "https://celestrak.org/SpaceData/SW-Last5Years.csv"
+            }
+        ],
+        "last_update": "2023-07-24T00:00:00.000",
+        "next_update_check": "2023-07-27T01:02:48.375049",
+        "check_frequency": "6 hours"
+    },
```

Comment:
We don't use that part here for now in here, right? Should we have `finals2000A` somewhere in the tests?